sql-plugin/src/main/scala/com/nvidia/spark/rapids/ParquetCachedBatchSerializer.scala

```diff
@@ -478,6 +481,10 @@ protected class ParquetCachedBatchSerializer extends GpuCachedBatchSerializer {
       originalSelectedAttributes: Seq[Attribute]): RDD[ColumnarBatch] = {
 
     val cbRdd: RDD[ColumnarBatch] = input.map {
+      case parquetCB: ParquetCachedBatch if parquetCB.sizeInBytes == 0 =>
+        // If the buffer is empty, we have cached an empty batch, we don't need to decode it,
+        // instead just return an empty ColumnarBatch
+        GpuColumnVector.emptyBatch(originalSelectedAttributes.asJava)
```

Comment:
How is this working?? `emptyBatch` produces a batch with columns but no rows.  But parquetCB.sizeInBytes will only be 0 if we had rows but no columns.  We are dropping the row data, right???